extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/GeoJsonSource.kt

```diff
@@ -312,21 +315,34 @@ class GeoJsonSource(builder: Builder) : Source(builder.sourceId) {
    * if data parsing error has occurred.
    *
    * Note: This method is not thread-safe. The Geometry is parsed on a worker thread, please make sure
-   * the Geometry is immutable.
+   * the Geometry is immutable as well as all collections that are used to build it.
```

Comment:
nit: shall we mention otherwise `MapboxConcurrentGeometryModificationException` will be thrown?

Reply:
it's mentioned in `@throws` a bit below, I believe it's enough